src/site/antora/modules/ROOT/examples/manual/markers/log4j2.xml

```diff
@@ -25,16 +25,14 @@
       <PatternLayout pattern="%d{HH:mm:ss.SSS} (%marker) %m%n"/>
     </Console>
   </Appenders>
+  <MarkerFilter marker="SQL"
```

Comment:
Really liked this `MarkerFilter` usage at the root level. :star_struck: I stole it for #2662.

Reply:
I introduced it to make the Logback example more complex/complete.